malasakit-django/pcari/templates/base_with_links.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
+{% blocktrans trimmed %}
```

Comment:
`blocktrans` is not available by default. This tag needs to be loaded with `{% load i18n %}`. Also, this change doesn't take effect since it doesn't override the `footnote` block.